metric_config_parser/metric.py

```diff
@@ -69,6 +75,9 @@ class Metric:
     type: str = "scalar"
     category: Optional[str] = None
     depends_on: Optional[List[Summary]] = None
+    owner: Optional[str] = None
```

Comment:
What happens to the metrics that have a list of strings for the `owner` field, e.g., https://github.com/mozilla/metric-hub/blob/b321e300a88c5ec8fe00ccb8a7c3a2668f85d931/definitions/firefox_desktop.toml#L299 ?

Reply:
Good catch. I'll push a fix to support these